Openstudio 3.6.0 rpm cannot be installed properly under CentOS 7.

Enhancement Request
--------------
<!--- Provide a general summary of the feature you would like to see implemented -->

## Detailed Description
Under `centos:7`
After I downloaded the rpm from the release page. Then install by
```
yum install -y openstudio.rpm 
```
will raise an exception for requires: /usr/local/bin/wish4.0.


```
[root@4cd97e37f9b7 Download]# yum install -y OpenStudio-3.6.0+860f5de185-CentOS-7.9.2009-x86_64.rpm
Loaded plugins: fastestmirror, ovl
Examining OpenStudio-3.6.0+860f5de185-CentOS-7.9.2009-x86_64.rpm: openstudio-3.6.0-3.6.0-1.x86_64
Marking OpenStudio-3.6.0+860f5de185-CentOS-7.9.2009-x86_64.rpm to be installed
Resolving Dependencies
--> Running transaction check
---> Package openstudio-3.6.0.x86_64 0:3.6.0-1 will be installed
--> Processing Dependency: /bin/csh for package: openstudio-3.6.0-3.6.0-1.x86_64
Loading mirror speeds from cached hostfile
 * base: mirror.web-ster.com
 * centos-sclo-rh: mirror.fcix.net
 * centos-sclo-sclo: mirror.umd.edu
 * epel: d2lzkl7pfhq30w.cloudfront.net
 * extras: mirror.siena.edu
 * updates: paducahix.mm.fcix.net
--> Processing Dependency: /usr/local/bin/wish4.0 for package: openstudio-3.6.0-3.6.0-1.x86_64
Error: Package: openstudio-3.6.0-3.6.0-1.x86_64 (/OpenStudio-3.6.0+860f5de185-CentOS-7.9.2009-x86_64)
           Requires: /usr/local/bin/wish4.0
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

If you run it with `--skip-broken` flag.

```
[root@4cd97e37f9b7 Download]# yum install -y OpenStudio-3.6.0+860f5de185-CentOS-7.9.2009-x86_64.rpm --skip-broken
```

It will not raise exception but will: 


```
---> Package libXau.x86_64 0:1.0.8-2.1.el7 will be installed
---> Package libXrender.x86_64 0:0.9.10-1.el7 will be installed
---> Package libpng.x86_64 2:1.5.13-8.el7 will be installed
---> Package openstudio-3.6.0.x86_64 0:3.6.0-1 will be installed
--> Processing Dependency: /usr/local/bin/wish4.0 for package: openstudio-3.6.0-3.6.0-1.x86_64
--> Running transaction check
---> Package dejavu-sans-fonts.noarch 0:2.33-6.el7 will be installed
--> Processing Dependency: dejavu-fonts-common = 2.33-6.el7 for package: dejavu-sans-fonts-2.33-6.el7.noarch
---> Package fontpackages-filesystem.noarch 0:1.44-8.el7 will be installed
---> Package openstudio-3.6.0.x86_64 0:3.6.0-1 will be installed
--> Processing Dependency: /usr/local/bin/wish4.0 for package: openstudio-3.6.0-3.6.0-1.x86_64
--> Running transaction check
---> Package dejavu-fonts-common.noarch 0:2.33-6.el7 will be installed
---> Package openstudio-3.6.0.x86_64 0:3.6.0-1 will be installed
--> Processing Dependency: /usr/local/bin/wish4.0 for package: openstudio-3.6.0-3.6.0-1.x86_64
--> Processing Dependency: /usr/local/bin/wish4.0 for package: openstudio-3.6.0-3.6.0-1.x86_64

Packages skipped because of dependency problems:
    dejavu-fonts-common-2.33-6.el7.noarch from base
    dejavu-sans-fonts-2.33-6.el7.noarch from base
    fontconfig-2.13.0-4.3.el7.x86_64 from base
    fontpackages-filesystem-1.44-8.el7.noarch from base
    freetype-2.8-14.el7_9.1.x86_64 from updates
    libX11-1.6.7-4.el7_9.x86_64 from updates
    libX11-common-1.6.7-4.el7_9.noarch from updates
    libXau-1.0.8-2.1.el7.x86_64 from base
    libXft-2.3.2-2.el7.x86_64 from base
    libXrender-0.9.10-1.el7.x86_64 from base
    libicu-50.2-4.el7_7.x86_64 from base
    libicu-devel-50.2-4.el7_7.x86_64 from base
    2:libpng-1.5.13-8.el7.x86_64 from base
    libxcb-1.13-1.el7.x86_64 from base
    openstudio-3.6.0-3.6.0-1.x86_64 from /OpenStudio-3.6.0+860f5de185-CentOS-7.9.2009-x86_64
    1:tcl-8.5.13-8.el7.x86_64 from base
    tcsh-6.18.01-17.el7_9.1.x86_64 from updates
    1:tk-8.5.13-6.el7.x86_64 from base
```
## Possible Implementation

- Rebuilt from CentOS with newer depedency meta-info
- Update CMakeList to catch up with the new wish dependency

```
-- Library backtrace found /root/.conan/data/libbacktrace/cci.20210118/_/_/package/1d877a3df840030e6a8abb74c5ffb9088d08b47a/lib/libbacktrace.a
-- Conan: Adjusting language standard
-- Current conanbuildinfo.cmake directory: /Openstudio/build
-- Conan: Compiler GCC>=5, checking major version 8
-- Conan: Checking correct version: 8
-- openstudio: DONE RUNNING CONAN
CMake Error at CMakeLists.txt:648 (message):
  EnergyPlus no longer provides packages for Ubuntu 18.04
```

If I tried to build it from scratch, raise exception with E+ compatitable issue.

Emmmm even after I use the `sed` it still failed:



````
[1235/9183] Building CXX object src/model/CMakeFi...rces.dir/ruby_OpenStudioModelResources_wrap.cxx.o
FAILED: src/model/CMakeFiles/ruby_OpenStudioModelResources.dir/ruby_OpenStudioModelResources_wrap.cxx.o
/opt/rh/devtoolset-10/root/usr/bin/g++ -DBOOST_STACKTRACE_ADDR2LINE_LOCATION=\"/usr/bin/addr2line\" -DBOOST_STACKTRACE_USE_ADDR2LINE -DBOOST_STACKTRACE_USE_BACKTRACE -DBOOST_STACKTRACE_USE_NOOP -DGEOGRAPHICLIB_SHARED_LIB=0 -DHAVE_BZIP2 -DHAVE_UNISTD_H -DLIBXML_STATIC -DLIBXSLT_STATIC -D_GLIBCXX_USE_CXX11_ABI=0 -D_NO_ASYNCRTIMP -D_NO_PPLXIMP -I/Openstudio/src -I/Openstudio/build/src -I/Openstudio -isystem /opt/rh/rh-python38/root/usr/include/python3.8 -isystem /root/.conan/data/openstudio_ruby/2.7.2/nrel/centos/package/9e91ea8bf0bb98e1b35847c4567cb0a3682d2648/include -isystem /root/.conan/data/openstudio_ruby/2.7.2/nrel/centos/package/9e91ea8bf0bb98e1b35847c4567cb0a3682d2648/include/ruby-2.7.0 -isystem /root/.conan/data/openstudio_ruby/2.7.2/nrel/centos/package/9e91ea8bf0bb98e1b35847c4567cb0a3682d2648/include/ruby-2.7.0/x86_64-linux -isystem /root/.conan/data/sqlite3/3.38.5/_/_/package/b7e8530a7ce4dc98890a185b197886c549602572/include -isystem /root/.conan/data/jsoncpp/1.9.5/_/_/package/82ef5eac51c38971dea2fd342dd55ddf2ddfbbc3/include -isystem /root/.conan/data/minizip/1.2.12/_/_/package/50479a03e6a0f0ad7e9baeddeb9019d627aa692f/include -isystem /root/.conan/data/minizip/1.2.12/_/_/package/50479a03e6a0f0ad7e9baeddeb9019d627aa692f/include/minizip -isystem /root/.conan/data/zlib/1.2.12/_/_/package/19729b9559f3ae196cad45cb2b97468ccb75dcd1/include -isystem /root/.conan/data/bzip2/1.0.8/_/_/package/91a8b22c2c5a149bc617cfc06cdd21bf23b12567/include -isystem /root/.conan/data/fmt/9.1.0/_/_/package/1d886ef72d27d7ab9a1a64be277b3d28a290f3e4/include -isystem /root/.conan/data/pugixml/1.12.1/_/_/package/82ef5eac51c38971dea2fd342dd55ddf2ddfbbc3/include -isystem /root/.conan/data/boost/1.79.0/_/_/package/a762cbb0f4f91f9ac21c111bfd58400aa5d97a0b/include -isystem /root/.conan/data/libbacktrace/cci.20210118/_/_/package/19729b9559f3ae196cad45cb2b97468ccb75dcd1/include -isystem /root/.conan/data/cpprestsdk/2.10.18/_/_/package/336065edd5db47193312b10d982fa47c72ef8211/include -isystem /root/.conan/data/openssl/1.1.1o/_/_/package/19729b9559f3ae196cad45cb2b97468ccb75dcd1/include -isystem /root/.conan/data/websocketpp/0.8.2/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/include -isystem /root/.conan/data/geographiclib/1.52/_/_/package/2066548edbbda32321eec39c17ddb5f988c7a268/include -isystem /root/.conan/data/libxml2/2.9.14/_/_/package/f9bf12b05a917e3b694bc35eb5037c71fbf6b7ca/include -isystem /root/.conan/data/libxml2/2.9.14/_/_/package/f9bf12b05a917e3b694bc35eb5037c71fbf6b7ca/include/libxml2 -isystem /root/.conan/data/libiconv/1.16/_/_/package/19729b9559f3ae196cad45cb2b97468ccb75dcd1/include -isystem /root/.conan/data/libxslt/1.1.34/_/_/package/d496057d7c19e1765bd39bf20d3383470a1603b0/include -fdiagnostics-color=always  -Wall -fPIC -fno-strict-aliasing -Winvalid-pch -Wnon-virtual-dtor -Wno-narrowing -Wno-narrowing -Werror -Wno-overloaded-virtual -Wno-maybe-uninitialized -O3 -DNDEBUG -std=c++2a -Wno-deprecated-declarations -Wno-sign-compare -Wno-register -Wno-conversion-null -Wno-misleading-indentation -fno-gnu-unique -MD -MT src/model/CMakeFiles/ruby_OpenStudioModelResources.dir/ruby_OpenStudioModelResources_wrap.cxx.o -MF src/model/CMakeFiles/ruby_OpenStudioModelResources.dir/ruby_OpenStudioModelResources_wrap.cxx.o.d -o src/model/CMakeFiles/ruby_OpenStudioModelResources.dir/ruby_OpenStudioModelResources_wrap.cxx.o -c /Openstudio/build/src/model/ruby_OpenStudioModelResources_wrap.cxx
g++: fatal error: Killed signal terminated program cc1plus
compilation terminated.
[1244/9183] Building CXX object src/model/CMakeFi...ModelHVAC.dir/ruby_OpenStudioModelHVAC_wrap.cxx.o
ninja: build stopped: subcommand failed.
````

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -773,7 +773,7 @@ if(UNIX)
     set(RADIANCE_PLATFORM "Darwin")
   elseif(EXISTS "/etc/redhat-release")
     # DLM: note that the installer expects the extracted directory to be named radiance-5.0.a.12-Linux
-    set(RADIANCE_EXPECTED_HASH f75898bbfd238a740c1902d385c42b92)
+    set(RADIANCE_EXPECTED_HASH 1e0ef7909c99cd353191d63b085dc6a9)
     set(RADIANCE_PLATFORM "Redhat")
   else()
     set(RADIANCE_EXPECTED_HASH 7b9e4df2b6050fe51c6e138e4c0e6a5a)

